File: WindowsAppVB1/Form1.vb
Public Class Form1
    Private Sub DemoHelloToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles DemoHelloToolStripMenuItem.Click
        Dim frm As New DemoHelloForm
        frm.MdiParent = Me
        frm.WindowState = FormWindowState.Maximized
        frm.Show()
        frm.Text = "DemoHelloForm"
    End Sub
End Class

File: WindowsFormsApp1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;


namespace WindowsFormsApp1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void demoHelloToolStripMenuItem_Click(object sender, EventArgs e)
        {
            Demo.DemoHelloForm frm = new Demo.DemoHelloForm();
            frm.MdiParent = this;
            frm.WindowState = FormWindowState.Maximized;
            frm.Show();
        }

        private void demoForm1ToolStripMenuItem_Click(object sender, EventArgs e)
        {
            Demo.DemoForm1 frm = new Demo.DemoForm1();
            frm.MdiParent = this;
            frm.WindowState = FormWindowState.Maximized;
            frm.Show();
        }
    }
}
